src/agent/README.md

```diff
@@ -139,6 +139,7 @@ The kata agent has the ability to configure agent options in guest kernel comman
 | `agent.server_addr` | Server address | Allow the ttRPC server address to be specified | string | `"vsock://-1:1024"` |
 | `agent.trace` | Trace mode | Allow to static tracing | boolean | `false` |
 | `systemd.unified_cgroup_hierarchy` | `Cgroup hierarchy` | Allow to setup v2 cgroups | boolean | `false` |
+| `agent.policy_file` | Policy file path | Allow to specify the policy file path for the agent to initialize policy engine with | string | `"/etc/kata-opa/default-policy.rego"` |
```

Comment:
I suggest: "Allow to specify the path to the default agent policy". 

Same for the config.rs comment below.

Reply:
Sure. I will change it.